src/Controller/CRUDController.php

```diff
@@ -149,10 +147,8 @@ public function listAction(Request $request)
      * Execute a batch delete.
      *
      * @throws AccessDeniedException If access is not granted
-     *
-     * @return RedirectResponse
      */
-    public function batchActionDelete(ProxyQueryInterface $query)
+    public function batchActionDelete(ProxyQueryInterface $query): RedirectResponse
```

Comment:
I kinda wonder if we should restrict to `RedirectResponse` or we should allow `Response`.